src/battle-animations.ts

```diff
@@ -2647,8 +2647,19 @@ export class PokemonSprite extends Sprite {
 	// Statbar
 	/////////////////////////////////////////////////////////////////////
 
+	getClassForPosition(slot: number) {
+		// DOUBLES: Slot0 -> left / Slot1 -> Right
+		// TRIPLES: slot0 -> left / Slot1 -> Center / Slot2 -> Right
+		const position = [
+			' left',
+			this.scene.activeCount === 3 ? ' center' : ' right',
+			' right',
```

Comment:
Maybe we should be using a different class name for these, `left`, `right`, and `center` are pretty generic terms that might get used elsewhere by mistake in the future.

Possibly `leftstatbar` or something similar? I'm open to discussion on this.

Reply:
You are right, so in the end it should be like this:

```suggestion
			' leftstatbar',
			this.scene.activeCount === 3 ? ' centerstatbar' : ' rightstatbar',
			' rightstatbar',
```